client-sdk/ts-web/rt/cypress/integration/playground.spec.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types="cypress" />
+
+describe('playground', () => {
+    it('finishes', () => {
+        cy.visit('http://localhost:8080/');
+        cy.contains('barolitopsis', {timeout: 60_000});
```

Comment:
Lol, could we put these into some (shared) constant?

Reply:
oh yeah, I'll see if we can

Reply:
we can! I'll do this in oasis-bridge too